src/ActionsMenu.scss

```diff
@@ -0,0 +1,33 @@
+.ActionsMenu__header {
+    position:relative;
+    background-color: #cecccc;
+    margin-right: 0;
+    padding-bottom: 0.05rem;
+    padding-top: 0.05rem;
+    text-align:center;
+}
+
+.ActionsMenu__header-heading {
+    font-size: inherit;
+    font-weight: bold;
+    padding-top: 0.1rem;
+}
+
+.ActionsMenu__menu {
+    background-color: #ffffff;
+    border-color: #cecccc;
+    border-style: none solid solid solid;
```

Comment:
Let's specify border width, so we know how much top position we need to adjust
`
border-width: 0.3rem;
`

Reply:
Oh, I forgot to mention, there seem to be a border at the top of ActionsMenu in the design, so I would change border-style to simply solid.
`
border-style: solid;
`

Reply:
Yes, that was left over from when it slid under the heading rather than off to the side, good catch.